qiita_ware/commands.py

```diff
@@ -66,34 +60,42 @@ def submit_EBI(preprocessed_data_id, action, send, fastq_dir_fp=None):
         # step 4: sending sequences
         old_ascp_pass = environ.get('ASPERA_SCP_PASS', '')
         environ['ASPERA_SCP_PASS'] = qiita_config.ebi_seq_xfer_pass
-        seqs_cmds = ebi_submission.generate_send_sequences_cmd()
+
         LogEntry.create('Runtime',
                         ("Submitting sequences for pre_processed_id: "
                          "%d" % preprocessed_data_id))
-        try:
-            # place holder for moi call see #1477
-            pass
-        except:
-            LogEntry.create('Fatal', seqs_cmds,
-                            info={'ebi_submission': preprocessed_data_id})
-        else:
-            LogEntry.create('Runtime',
-                            ('Submission of sequences of pre_processed_id: '
-                             '%d completed successfully' %
-                             preprocessed_data_id))
+        for cmd in ebi_submission.generate_send_sequences_cmd():
+            cmd_pieces = shsplit(cmd)
+            try:
+                call(cmd_pieces, stdout=open(ebi_submission.ascp_reply, 'a'))
+            except:
+                with open(ebi_submission.ascp_reply, 'r') as f:
+                    content = f.read()
+                ebi_submission.study.ebi_submission_status = (
+                    "failed: ASCP - %s" % content)
+                LogEntry.create('Fatal', content,
+                                info={
+                                    'ebi_submission': preprocessed_data_id,
+                                    'fail': cmd})
+        LogEntry.create('Runtime',
+                        ('Submission of sequences of pre_processed_id: '
+                         '%d completed successfully' %
```

Comment:
So the lack of fireworks is the celebration of success, as this is real
success? ... so postmodern.

On (Oct-09-15|10:19), Antonio Gonzalez wrote:
> > +        for cmd in ebi_submission.generate_send_sequences_cmd():
> > +            cmd_pieces = shsplit(cmd)
> > +            try:
> > +                call(cmd_pieces, stdout=open(ebi_submission.ascp_reply, 'a'))
> > +            except:
> > +                with open(ebi_submission.ascp_reply, 'r') as f:
> > +                    content = f.read()
> > +                ebi_submission.study.ebi_submission_status = (
> > +                    "failed: ASCP - %s" % content)
> > +                LogEntry.create('Fatal', content,
> > +                                info={
> > +                                    'ebi_submission': preprocessed_data_id,
> > +                                    'fail': cmd})
> > +        LogEntry.create('Runtime',
> > +                        ('Submission of sequences of pre_processed_id: '
> > +                         '%d completed successfully' %
> 
> Nop, cause this time it actually works well (or at least we hope!) vs. giving you the impression via fireworks that it does. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1488/files#r41654914